webf/lib/src/svg/rendering/shape.dart

```diff
@@ -67,6 +105,36 @@ abstract class RenderSVGShape extends RenderBoxModel {
 
   Path asPath();
 
+  void parseDefs(Rect rect) {
+    var element = renderStyle.target as SVGElement;
+    dynamic fillAttr = element.attributeStyle['fill'] ?? findRoot()?.element.attributeStyle['fill'];
+    dynamic clipPathAttr = element.attributeStyle['clipPath'] ?? findRoot()?.element.attributeStyle['clipPath'];
+    if (fillAttr == null && clipPathAttr == null) return null;
```

Comment:
should not return null , just return is better ? https://dart.dev/tools/linter-rules/avoid_returning_null_for_void